### ISSUE TITLE
Double selector for input navigation

### ISSUE THREAD

**Issue by asvae** [epicmaxco/vuestic-ui#3530]
**Vuestic-ui version:** 1.6.6
<img width="488" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/7e2f0e34-f76f-4894-94e7-792844602fda">

For some form components when using keyboard navigation you select first input, then icon. There should be only input selection.

**Comment by zvenigorodskaia** [comment]
https://ui-develop.vuestic.dev/ui-elements/time-input#default

please change behavior for default Time input (same with default date input) - click in the input shold open dropdown (not only click by icon)
